Sources/CryptoSwift/SHA2.swift

```diff
@@ -316,8 +316,6 @@ extension SHA2: Updatable {
             self.process32(block: chunk, currentHash: &self.accumulatedHash32)
           case .sha384, .sha512:
             self.process64(block: chunk, currentHash: &self.accumulatedHash64)
-          @unknown default:
-            preconditionFailure()
```

Comment:
<img width="1066" alt="Screenshot 2021-06-18 at 11 58 00" src="https://user-images.githubusercontent.com/758033/122543788-72514800-d02c-11eb-9268-2770ce73d1b6.png">

It doesn't look right. It is needed to silence the warning